src/plugins/unified_field_list/common/index.ts

```diff
@@ -7,3 +7,4 @@
  */
 
 export const PLUGIN_ID = 'unifiedFieldList';
+export const FIELD_EXISTENCE_SETTING = 'unifiedFieldList:useFieldExistenceSampling';
```

Comment:
I think need migration from `lens:useFieldExistenceSampling` to  `unifiedFieldList:useFieldExistenceSampling`, else the previous value of `unifiedFieldList:useFieldExistenceSampling` will get lost